src/components/menu/SubMenu.tsx

```diff
@@ -0,0 +1,32 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import { OpenAction, IMenuPositionConfig } from './MenuSharedTypes';
+import MenuState, { IMenuStateConfig } from './MenuState';
+import MenuOpener from './MenuOpener/MenuOpener';
+
+export interface IMenuProps extends Partial<IMenuStateConfig>, Partial<IMenuPositionConfig> {
+    openAction?: OpenAction;
+
+    toggler: React.ReactNode;
+    children: React.ReactNode;
+}
+
+const Menu = (props: IMenuProps) => (
```

Comment:
Component should match name of file. :) 

This should be `SubMenu`

Reply:
: React.SFC<ISubMenuProps>

Reply:
Renamed to SubMenu, added missing type.